pkg/render/node.go

```diff
@@ -57,29 +55,32 @@ var (
 // Node creates the node daemonset and other resources for the daemonset to operate normally.
 func Node(
 	k8sServiceEp K8sServiceEndpoint,
-	cr *operator.Installation,
+	cr operator.InstallationSpec,
```

Comment:
I'm not sure about switching these from pointers. Doesn't that mean a shallow copy will be done.